contrib/platform/test/com/sun/jna/platform/win32/COM/util/IDispatchTest.java

```diff
@@ -107,9 +110,9 @@ public void testDispatchBaseOnNamed() throws InterruptedException {
             }
         }
 
-        TestCase.assertTrue(navigationHappend);
-        
         ieApp.Quit_MOD();
+        
+        TestCase.assertTrue(navigationHappend);
     }
```

Comment:
Use _org.junit.Assert.assertXXX_ the _TestCase_ ones (which are actually from _junit.framework_) are **deprecated**.